app/javascript/admin/adminModal.js

```diff
@@ -31,7 +35,7 @@ export const adminModal = function ({
       <div class="crayons-modal__box">
         <header class="crayons-modal__box__header">
           <p class="fw-bold fs-l">${title}</p>
-          <button type="button" class="crayons-btn crayons-btn--icon crayons-btn--ghost" data-action="click->config#closeAdminModal">
```

Comment:
I repurposed the `adminModal.js` file slightly in order to make it usable within other stimulsJS controllers (not JUST the `config` controller)

Reply:
I'm not sure all of the places where this modal is currently used, but it's not accessible 🙁  (missing some roles, focusTrap etc). Is it possible to refactor this to use the Preact `<Modal />` component so that we can take advantage of all the baked-in accessibility behaviours?

Reply:
After completing this PR, I plan to refactor places within `Admin` that currently use this un-a11y `adminModal.js`. This file will eventually go away.